__tests__/app.test.js

```diff
@@ -420,6 +420,84 @@ describe('CORE: GET /api/articles/:article_id/comments', () => {
     })
 })
 
+describe('CORE: POST /api/articles/:article_id/comments', () => {
+    test('returns a status code of 201 and an object', () => {
```

Comment:
we could also check whether the id is correct (should be the next id compared to what the last comment is in the db)